pkg/subscriptions/get_monotonic_start.py

```diff
@@ -0,0 +1,24 @@
+import ctypes, os
+
+# see <linux/time.h>
+CLOCK_REALTIME = 0
+CLOCK_MONOTONIC = 1
+
+class timespec(ctypes.Structure):
+    _fields_ = [
+        ('tv_sec', ctypes.c_long),
+        ('tv_nsec', ctypes.c_long)
+    ]
+
+librt = ctypes.CDLL('librt.so.1', use_errno=True)
+c_clock_gettime = librt.clock_gettime
+c_clock_gettime.argtypes = [ctypes.c_int, ctypes.POINTER(timespec)]
+
+def clock_gettime(clock):
+    t = timespec()
+    if c_clock_gettime(clock , ctypes.pointer(t)) != 0:
+        errno_ = ctypes.get_errno()
+        raise OSError(errno_, os.strerror(errno_))
+    return t.tv_sec + t.tv_nsec * 1e-9
+
+print clock_gettime(CLOCK_REALTIME) - clock_gettime(CLOCK_MONOTONIC)
```

Comment:
You use this with `python.spawn()`, which will prefer Python 3. You can install Py 3 on RHEL 7 (even though it takes external repos), so let's please add some () around this to make it bilingual.

Reply:
It prefers platform-python, which is Python 2 on RHEL 7.  I think we can rely on that.

(In fact, the rhel-7-8 image has Python 3 installed, and this script still works.  But the existing tests wouldn't have caught a breakage, and I have added a check for that.)

Reply:
And I add the `()` as well!